sql/core/src/main/scala/org/apache/spark/sql/jdbc/DB2Dialect.scala

```diff
@@ -20,13 +20,26 @@ package org.apache.spark.sql.jdbc
 import java.sql.Types
 import java.util.Locale
 
+import org.apache.spark.sql.connector.expressions.aggregate.{AggregateFunc, GeneralAggregateFunc}
 import org.apache.spark.sql.types._
 
 private object DB2Dialect extends JdbcDialect {
 
   override def canHandle(url: String): Boolean =
     url.toLowerCase(Locale.ROOT).startsWith("jdbc:db2")
 
+  override def compileAggregate(aggFunction: AggregateFunc): Option[String] = {
+    super.compileAggregate(aggFunction).orElse(
+      aggFunction match {
+        case f: GeneralAggregateFunc if f.name() == "VAR_POP" =>
```

Comment:
this is the only one DB2 supports?

Reply:
I tests other function and throw error that tell me DB2 doesn't support them.
According to the latest doc of DB2, it supports `STDDEV_POP` too.
It seems the docker image uses some lower version of DB2.

Reply:
I thought both `STDDEV_POP` and `STDDEV_SAMP` are supported in DB2 version 11.5. Docker image is 11.5.6.0a?

Reply:
It seems Docker image con contains the latest code.